charts/keess/templates/cluster-role.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "keess.fullname" . }}
+  labels:
+    {{- include "keess.labels" . | nindent 4 }}
+rules:
+- apiGroups: [""]
+  resources:
+  - configmaps
+  - secrets
+  verbs: ["get", "create", "patch", "delete", "list", "watch"]
+- apiGroups: [""]
+  resources:
+  - namespaces
+  verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources:
+  - nodes
+  verbs: ["list"]
+- apiGroups: [""]
+  resources:
+  - events
+  verbs: ["create"]
+{{- end }}
```

Comment:
Same here. Move to PAC? @iblackman @bcdonadio ?

Reply:
Same before

Reply:
The chart has the logic to not generated those if desired, which I consider correct. The proper way to deploy it in PAC can be addressed on the PR to deploy it in PAC.

Reply:
Since he added the option to not generate those it works for both scenarios. Now how we will use this chart to deploy in pac is a discussion that we need to have in the PR in pac once we have it. For the chart it is good to have it as more generic and customizable as possible.